data/index.json

```diff
@@ -177,11 +177,12 @@
     "name": "big-decent-clipped-38",
     "type": "table",
     "id": "big-decent-clipped-38",
+    "separator": ",",
     "description": "",
     "creator": "Alexander Lex",
     "path": "data_38.csv",
     "size": [
-      19,
+      102,
```

Comment:
It was. which is why I'm not sure why Sam said the data desc couldn't be created automatically.